Pod/Classes/NBNRealmBrowser.m

```diff
@@ -8,6 +8,16 @@
 
 static NSString *CellIdentifier = @"CellIdentifier";
 
+@interface RLMRealm ()
++ (instancetype)realmWithPath:(NSString *)path
```

Comment:
How likely is it to change? Because the [removed headers](https://github.com/nerdishbynature/NBNRealmBrowser/pull/3) also came all of a sudden. What value does it add?